### DISCUSSION TITLE
Chia files location Ubuntu GUI binary

### DISCUSSION THREAD

**Discussion by andymarden** [martomi/chiadog#377] [General]
Hi

Struggling with the new prompt that appears with the latest chia version. It is asking for where the venv is. I have installed the GUI binary for ubuntu and I don't know where that is nor can I find it.

Any thoughts?

Cheers
Andy

**Comment by andymarden** [comment]
Sorry - ignore - got mixed up between farmr and chiadog

**Comment by bradyjarvis** [comment]
Hello @andymarden,
Also worth pointing out that Farmr has been discontinued.

However , all is not lost.  There is an official chia farming dashboard I believe.  I haven't tried it yet, but JM covered it here:
https://youtu.be/7NNbmuwf99s
